contributors/guide/first-contribution.md

```diff
@@ -19,70 +19,98 @@ description: |
-The `good first issue` label indicates that members have committed to providing [extra assistance](/contributors/guide/help-wanted.md) for new contributors.
+There are [multiple repositories] within the Kubernetes organization.
+Each repository has beginner-friendly issues that are a great place to 
+get started on your contributor journey. For example, [kubernetes/kubernetes] has 
+[help wanted] and [good first issue] labels for issues that don't need high-level 
+Kubernetes knowledge to contribute to. The `good first issue` label also indicates 
+that Kubernetes Members have committed to providing [extra assistance] for new 
+contributors. Another way to get started is to find a documentation improvement, 
+such as a missing/broken link, which will give you exposure to the code 
+submission/review process without the added complication of technical depth. 
 
-Another good strategy is to find a documentation improvement, such as a missing/broken link, which will give you exposure to the code submission/review process without the added complication of technical depth. Please see [Contributing](#contributing) below for the workflow.
+### Issue Assignment in Github
 
-#### Issue Assignment in Github
+When you've found an issue to work on, you can assign it to yourself.
 
-When you are willing to take on an issue, you can assign it to yourself. Just reply with `/assign` or `/assign @yourself` on an issue, 
-then the robot will assign the issue to you and your name will present at `Assignees` list.
+* Reply with `/assign` or `/assign @yourself` on the issue you'd like to work on 
+* The [K8s-ci-robot] will automatically assign the issue to you. 
+* Your your name will then be listed under, `Assignees`.
 
-### Learn about SIGs
+## Learn about SIGs
 
-You may have noticed that some repositories in the Kubernetes Organization are owned by [Special Interest Groups](/sig-list.md), or SIGs.
-We organize the community into SIGs in order to improve our workflow and more easily manage what is a very large community project.
-The developers within each SIG have autonomy and ownership over that SIG's part of Kubernetes. Check out the [list of SIGs](/sig-list.md) for contact information.
+Some repositories in the Kubernetes Organization are owned by 
+[Special Interest Groups], or SIGs.
 
-Understanding how to interact with SIGs is an important part of contributing. 
+The Kubernetes community is broken out into SIGs in order to improve its workflow,
+and more easily manage what is a very large community project. The developers 
+within each SIG have autonomy and ownership over that SIG's part of Kubernetes. 
+Understanding how to interact with SIGs is an important part of contributing 
+to Kubernetes. Check out the [list of SIGs] for contact information.
```

Comment:
```suggestion
to Kubernetes. Check out the [list of SIGs][sl] for contact information.
```
Just noticed this when viewing it in actual markdown mode, adding `[sl]` should bring it in line with the rest of the other links.